src/cloudant/account.py

```diff
@@ -221,14 +201,14 @@ def delete_database(self, dbname):
 
     def db_updates(self, since=None, continuous=True):
         """
-        _db_updates_
-
-        Implement streaming from _db_updates feed. Yields information about
-          databases that have been updated.
+        Streams data from _db_updates feed. Yields information about
+        databases that have been updated.
 
-        :param str since: Start from this sequence
-        :param boolean continuous: Stream results
+        :param str since: Starts update streaming from this sequence
```

Comment:
I've gone ahead and attempted to make all of the parameter descriptions full sentences and made sure that they all end in a period.  For "return" descriptions I've decided to leave them as sentence fragments since they do not lend themselves easily to being sentences due to the fact that they are all prefaced with "Returns:" when the documentation is rendered.  So the "return" descriptions are not ended with a period.  This has been done throughout the library.